nagios.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"fmt"
+	"os/exec"
+)
+
+// GetNagiosStatusData is a task factory for tasks that fetch a JSON representation of
+// the provided project. The task uses outFor and errOutFor to get io.Writers
+// to write, respectively, the JSON output and any eventual error message.
+func GetNagiosStatusData(project, pod string, outFor, errOutFor projectResourceWriterCloserFactory) Task {
+	return retrieveNagiosData(func() *exec.Cmd {
+		return exec.Command("oc", "exec", pod, "--", "cat", "/var/log/nagios/status.dat")
```

Comment:
@philbrookes shouldn't we be specifying also the container name here?

Came across this when refactoring, and wondering if it's not hiding the same situation with hit with logs.